castle/cms/tests/test_archival.py

```diff
@@ -23,6 +22,9 @@
 from plone.app.testing import setRoles
 
 
+install_aliases()  # noqa
```

Comment:
this should be left in it's previous position under the import directly so that it can correctly override imports with aliases from future.standard_library